src/_data/side-nav.yml

```diff
@@ -280,24 +277,21 @@
 - title: Resources
   expanded: false
   children:
-    - title: Overview
-      urlExactMatch: true
-      match-page-url-exactly: true
-      permalink: /resources
-    - title: Books
-      permalink: /resources/books
-    - title: "DartPad in tutorials: Best practices"
-      permalink: /resources/dartpad-best-practices
-    - title: Code of conduct
-      permalink: /code-of-conduct
     - title: FAQ
       permalink: /faq
-    - title: History
+    - title: Language evolution
+      permalink: /guides/language/evolution
+    - title: Language specification
+      permalink: /guides/language/spec    
+    - title: Coming from ...
```

Comment:
**question:** Would this be more at home under **Development**?

Reply:
hmm, that's an idea.. from the original PR description:
> _temporarily move to Resources, will go under new "Learn Dart" section_

I'm picturing it along side other learning guidance, like getting started and tutorials, etc. So, keeping it under resources is only temporary until I start the re-work of the "Samples & tutorials" umbrella (soon-to-be "Learn Dart"), but maybe then we should revisit where these belong. 

(I feel like "Development" is a little ill-defined, for myself at least, so I'm hesitant to place more content under there atm)

Reply:
It is rather ill-defined, especially now we're moving more things out of **Language** under **Development**, but so far everything under there is about learning the specifics of a specific concept, unrelated to the core language itself. These aren't a specific concept, so I think **Resources** is fine in the mean time.

Reply:
OK, let's consider this a "Phase 2" question at this point. We can circle back.